package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://bitbucket.org/yooapps/yoo-digital-typescript.git"
+    "url": "https://github.com/yoo-digital/typescript.git"
```

Comment:
Why not use ssh URLs like in the css package?

Reply:
Good catch, adjusted!